src/Compilers/CSharp/Test/Semantic/Semantics/PropertyFieldKeywordTests.cs

```diff
@@ -1350,9 +1378,394 @@ public S(int arg)
+            }
+
+            Assert.Equal(0, accessorBindingData.NumberOfPerformedAccessorBinding);
+
+            if (fieldAccessorFirst)
+            {
+                Assert.Equal("System.Double C.<P>k__BackingField", comp.GetTypeByMetadataName("C").GetFieldsToEmit().Single().ToTestDisplayString());
+            }
+            else
+            {
+                Assert.Empty(comp.GetTypeByMetadataName("C").GetFieldsToEmit());
+            }
+        }
+
+        [Theory]
+        [InlineData("get => 0; get => field;")]
+        [InlineData("get => field; get => 0;")]
+        [InlineData("get { return 0; } get => field;")]
+        [InlineData("get => field; get { return 0; }")]
+        [InlineData("get => 0; get { return field; }")]
+        [InlineData("get { return field; } get => 0;")]
+        [InlineData("get { return 0; } get { return field; }")]
+        [InlineData("get { return field; } get { return 0; }")]
+        public void SpeculativeSemanticModel_FieldInDuplicateAccessorWhereFirstAccessorUsesField_BindSpeculatedFirst(string accessors)
+        {
+            var comp = CreateCompilation($@"
+class C
+{{
+    public double P
+    {{
+        {accessors}
+    }}
+}}
+");
+            var accessorBindingData = new SourcePropertySymbolBase.AccessorBindingData();
+            comp.TestOnlyCompilationData = accessorBindingData;
+
+            var tree = comp.SyntaxTrees[0];
+            var token = tree.GetRoot().DescendantTokens().Single(t => t.IsKind(SyntaxKind.NumericLiteralToken));
+
+            var model = comp.GetSemanticModel(tree);
+            var identifier = SyntaxFactory.ParseExpression("field");
+            model.TryGetSpeculativeSemanticModel(token.SpanStart, (IdentifierNameSyntax)identifier, out var speculativeModel);
+            var fieldKeywordSymbolInfo = speculativeModel.GetSymbolInfo(identifier);
+
+            Assert.Equal("System.Double C.<P>k__BackingField", comp.GetTypeByMetadataName("C").GetFieldsToEmit().Single().ToTestDisplayString());
+            Assert.Equal("System.Double C.<P>k__BackingField", fieldKeywordSymbolInfo.Symbol.ToTestDisplayString(includeNonNullable: true));
+            Assert.Equal(0, accessorBindingData.NumberOfPerformedAccessorBinding);
+        }
+
+        [Theory]
+        [InlineData("get => 0; get => 1;", 19, 0)]
+        [InlineData("get => 0; get => 1;", 19, 1)]
+        [InlineData("get { return 0; } get => 1;", 27, 0)]
+        [InlineData("get { return 0; } get => 1;", 27, 1)]
+        [InlineData("get => 0; get { return 1; }", 19, 0)]
+        [InlineData("get => 0; get { return 1; }", 19, 1)]
+        [InlineData("get { return 0; } get { return 1; }", 27, 0)]
+        [InlineData("get { return 0; } get { return 1; }", 27, 1)]
+        public void SpeculativeSemanticModel_FieldInDuplicateAccessorWhereFirstAccessorDoesNotUseField_BindOriginalFirst(string accessors, int diagnosticColumn, int numericLiteralToSpeculate)
```

Comment:
>FieldInDuplicateAccessorWhereFirstAccessorDoesNotUseField [](http://example.com/codeflow?start=45&length=57)

> FieldInDuplicateAccessorWhereFirstAccessorDoesNotUseField

Is this accurate? Is ```[InlineData("get => 0; get => 1;", 19, 0)]``` really binds within a duplicate accessor?

Reply:
> I don't see what's wrong with the new suggested name. Can you elaborate please?

"**One**DoesNotUseField", but it looks like `field` keyword isn't used anywhere.